Makefile

```diff
@@ -1,4 +1,6 @@
 TARGETS := $(shell ls scripts | grep -v \\.sh)
+GO_FILES ?= $$(find . -name '*.go' | grep -v vendor | grep -v generated)
```

Comment:
nit: we don't have a `vendor` folder for k3s

Reply:
if someone use `make deps` it generates one